### ISSUE TITLE
vmui sends 3 requests to tsdb instead of 2

### ISSUE THREAD

**Issue by rtm0** [VictoriaMetrics/VictoriaMetrics#6240]
### Describe the bug

After I pick a day using a calendar widget, vmui sends 3 requests to tsdb. For example, if I pick `2024-05-07`, the following requests are sent:

```
GET /prometheus/api/v1/status/tsdb?topN=10&date=2024-05-07
GET /prometheus/api/v1/status/tsdb?topN=10&date=2024-05-05
GET /prometheus/api/v1/status/tsdb?topN=0&date=2024-05-07
```

Per discusstion with @valyala, only the first two should be sent.

Additionally, the date in the second request is two days before the chosen date. Not sure if this is a bug, but it feels like it should be just one day before? I.e. in the example above, it should be `2024-05-06` instead fo `2024-05-05`.

### To Reproduce

1. Navigate to `Explore > Explore Cardinality` and open the browser's developer console and choose `Network` tab
2. Click on the date in the right upper corner of the vmui and pick a day
3. Confirm vmui send 3 requests:  ![vmui-3-requests](https://github.com/VictoriaMetrics/VictoriaMetrics/assets/149964189/079e0447-2a5a-4b3f-a641-50a58611e292)
 

### Version

`v1.100.1` built locally with `make victoria-metrics`

**Comment by Loori-R** [comment]
Thank you for describing the problem.
The third request is needed to obtain totals when using the "Time series selector" filter; if it is empty, then the request is redundant.
The date should be set to one day back, it seems to be related to #6239

**Comment by hagen1778** [comment]
Related ticket https://github.com/VictoriaMetrics/VictoriaMetrics/issues/5231

**Comment by valyala** [comment]
The issue should be fixed in [v1.97.5 LTS release](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.97.5) and in [v1.102.0-rc1 release](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.102.0-rc1).
